src/Components/Accounts/MnemonicAndHdPath.js

```diff
@@ -1,18 +1,32 @@
 import React, { PureComponent } from 'react'
+import MnemonicInfoModal from './MnemonicInfoModal'
+import OnlyIf from '../../Elements/OnlyIf'
 
 export default class MnemonicAndHdPath extends PureComponent {
-  shouldComponentUpdate (nextProps, nextState) {
-    return (
-      nextProps.mnemonic !== this.props.mnemonic ||
-      nextProps.hdPath !== this.props.hdPath
-    )
```

Comment:
This looks like cruft that's no longer useful, and it prevents the modal from showing unless more conditions are added to that expression. I don't below `mnemonic` and `hdPath` will change at all except perhaps on initialization. My only guess was that this was included to save on render times. If this is an issue we can move the conditional to the `render` method.